### PR TITLE
[SD-3229] - Real time currency changes

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

File: server/macros/currency_base.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    return float(r.text.split(',')[1])
+
+
+def do_conversion(item, rate, prefix, search_param):
+    """
+    Performs the conversion
+    :param item: story
+    :param rate: exchange rate
+    :param prefix: to be used in the results
+    :param search_param: search parameter to locate the original value
```
Comment (plamut) [inline server/macros/currency_base.py]: IMO it should be made a bit more clear that `search_param` should, in fact, be a valid regular expression pattern, and not just an arbitrary string, for example.

**Comment by amagdas** [comment]
:+1:

**Pull request merged**